impl/src/main/java/org/apache/myfaces/el/ELResolverBuilder.java

```diff
@@ -159,35 +157,19 @@ protected boolean isReplaceImplicitObjectResolverWithCDIResolver(FacesContext fa
         BeanManager beanManager = CDIUtils.getBeanManager(facesContext);
         if (beanManager != null)
         {
-            FacesConfigBeanHolder holder = CDIUtils.get(beanManager, FacesConfigBeanHolder.class);
```

Comment:
like discussed in JIRA, i would not touch this for now
we should create a extra issue + discussion